### ISSUE TITLE
About Inputs argus comment in "metric.py"

### ISSUE THREAD

**Issue by LittlefatChuan** [btxviny/Trajectory-Optimization-and-Parametric-filtering-based-Video-Stabilization#1]
Thank you for personal implementation, and other awesome job, that helps me a lot.
Just a little question about your comment in metrics.py:
`    ''' Inputs: 
            path1: path to ground truth stable video
            path2: path to generated stable video
        Outputs:
            cropping_score
            distortion_score
            pixel_score
            stability_score
    '''`
i guess path 1 should be the path of unstable video instead of ground truth, right?

**Comment by btxviny** [comment]
Yes you're absolutely right. These evaluation metrics are non-reference, which means we have no ground truth stable video, to compare the result to. Sorry for the mistake.
